.changeset/loud-keys-give.md

```diff
@@ -0,0 +1,8 @@
+---
+"@latticexyz/entrykit": patch
+---
+
+- Move back to login flow modal after top-up.
+- Select default chain as source chain if connected chain is not part of selectable chains.
+- Make "Switch chain" button primary color to make sure it appears clickable.
+- Update successful deposit status message.
```

Comment:
Nit: let's phrase these in present tense or past tense for the changeset instead of the imperative commit message style. (I.e. "Entrykit now returns to the login flow after a successful top up.")

Reply:
updated, it does sound better now imo